plugins/backfill/transformerExporter.go

```diff
@@ -0,0 +1,52 @@
+// This is a plugin generated to export the configured transformer initializers
+
+package main
+
+import (
+	deny "github.com/makerdao/vdb-mcd-transformers/transformers/events/auth/deny_initializer"
+	rely "github.com/makerdao/vdb-mcd-transformers/transformers/events/auth/rely_initializer"
```

Comment:
Also I was trying to think if there could be a way to configure the backfill command so that we wouldn't have to redeploy if we discover that another event also needs to be backfilled. 🤔 It seems that we'll need to run backfills when new collaterals are deployed, so we'd need to update the config file with the new collateral contracts which would trigger a redeploy anyway, so it's probably not a big deal. 🤷

Reply:
Yeah, definitely a bit of a bummer to need to update this whenever we have something new that needs to be backfilled, but I couldn't really think of a way around that if we're in a situation where we truly need to check many blocks for a new event 🤔 

I am hoping that we can run this container in (potentially several) disposable resources, so that the next time we update this config we could wipe out everything that's already here (since it's being taken care of in an active backfill process) and include _only_ the new transformers requiring backfill (to spin up a new backfill process for that one).